reference/render/group-confirm--default.html

```diff
@@ -1,7 +1,7 @@
 <div class="form-group dso-confirm">
   <div class="dso-field-container">
     <div class="dso-selectable">
-      <input type="checkbox" id="input-confirm" name="" value="" />
+      <input type="checkbox" id="input-confirm" name="input-confirm" value="" />
```

Comment:
Jammer dat we in de nunjucks `value="{{  value }}"` hebben staan terwijl er geen een configuratie is waar `value` ook echt bestaat

Reply:
value verwijderd